Clean IokiService calling IokiApi

StefMa opened the pull request:
We have the problem that for `calculateNewFareForRide` and `updatePassengersForRide` the rideId and acccessToken were swapped 🫣 .

https://github.com/ioki-mobility/kmp-passenger-api/blob/86686e21b9e6ca4c55c203bfc79ae5de1f678ebd/library/src/commonMain/kotlin/com/ioki/passenger/api/IokiService.kt#L498-L507

https://github.com/ioki-mobility/kmp-passenger-api/blob/86686e21b9e6ca4c55c203bfc79ae5de1f678ebd/library/src/commonMain/kotlin/com/ioki/passenger/api/IokiService.kt#L509-L525

This PR fixes it.

To make sure this doesn't happen again (and not adding a bunch of tests for this) I introduced named arguments everywhere.

However, as another step, we could also think about introducing a `value class AccessToken(token: String)`... 🤔 

Besides that I also align our code formatting here. Its not consistent 🙃

juschmitt reviewed (commented):
You forgot `getBootstrap()` in line `550`.

About the value class, I do like the idea. adding named arguments makes it a bit more obvious that its wrong, but because rideId and accessToken are both Strings it would still "work".
Adding an actual type for accessToken makes it less possible to get wrong.

StefMa commented:
Fixed `bootstrap`, thanks!

Regarding the `value class`, its a bit more todo.
Also this might be general imrovemnt so I extracted it.
See https://github.com/ioki-mobility/kmp-passenger-api/pull/94

I'm going to merge this PR now.
